src/raftstore/store/fsm/peer.rs

```diff
@@ -121,6 +123,13 @@ impl<T, C> Store<T, C> {
     }
 }
 
+// For now, we just need to reclaim data and context
+#[inline]
+fn handle_entry_gc(e: Entry, gc_vec: &mut EntryGcKey) {
```

Comment:
It seems it's better to name it `handle_entry_cache_gc`?

Reply:
ok